mkosi.conf.d/20-debian/mkosi.conf

```diff
@@ -28,6 +28,7 @@ Packages=
         libqrencode4
         linux-image-generic
         linux-perf
+        linux-sysctl-defaults
```

Comment:
Should we maybe be doing this in mkosi proper for all bootable Debian images? It only got added in trixie, so we haven't considered this yet.

Reply:
It might affects things like the network tests, so I'd much rather not, people who want to use it can just pull it in via their confs, like here